man/networkd.conf.xml

```diff
@@ -70,6 +70,14 @@
         is false. Defaults to yes.</para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>RouteTable=</varname></term>
+        <listitem><para>Specifies the route table name. Takes a route name and table number separated with a colon.
+        (<literal><replaceable>name</replaceable>:<replaceable>integer</replaceable></literal>. The route table number
+        must be an integer in the range 1..4294967295. This setting can be specified multiple times. If an empty string
+        is specified, then all options specified earlier are cleared. Defaults to unset.</para></listitem>
```

Comment:
I'd like to this accepts a white-space separated list of name:integer pairs, eg `RouteTable=hoge:123 foo:456`.

Reply:
This is OK for few tables but what about `4294967295`  number of tables and imposible to write in one line . Hence This is this way

Reply:
Of course, the setting should _also_ accept multiple assignment, e.g.
```
RouteTable=hoge:123 foo:456
RouteTable=aaa:1111 bbb:2222 ccc:3333
```

Reply:
It's too complex . Let's just stick to same as dhcp options much easier to manage for the user also.

Reply:
I do not think so, e.g. `SystemCallFilter=`. But ok, let's extend that in another PR.